lib/hanami/logger.rb

```diff
@@ -142,7 +146,7 @@ def _message_hash(message)
       # @since x.x.x
       # @api private
       def _format(hash)
-        hash.map { |k, v| "#{k}=#{v}" }.join(SEPARATOR)
+        hash.map { |k, v| "#{k}=#{v}" unless v.nil? }.join(SEPARATOR) + NEW_LINE
```

Comment:
Yes, why `nil` values would be passed here? And in that case, it would be probably good to keep the key.

```ruby
{a: nil, b: 1}.map { |k, v| "#{k}=#{v}" }.join(" ")
# => "a= b=1"
```

@joneslee85 @davydovanton wdyt?